site/content/docs/4.3/utilities/api.md

```diff
@@ -13,6 +13,7 @@ The `$utilities` map contains all utilities and is later merged with your custom
 
 - `property`: Name of the property, this can be a string or an array of strings (needed for eg. horizontal paddings or margins).
 - `responsive` _(optional)_: Boolean indicating if responsive classes need to be generated. `false` by default.
+- `rfs` _(optional)_: Variable to enable fluid rescaling. Have a look at the [RFS]({{< docsref "/getting-started/rfs" >}}) page to find out how this works. `false` by default.
```

Comment:
`$enable-rfs` default value is set to `true`. So we have to adapt the boolean value in the docs as well. Maybe change the sentence to something like `Variable to disable fluid rescaling [...]` and change default value accordingly.